internal/interpreter/interpreter_error.go

```diff
@@ -220,3 +220,11 @@ type InvalidAccountName struct {
 func (e InvalidAccountName) Error() string {
 	return fmt.Sprintf("Invalid account name: @%s", e.Name)
 }
+
+type InvalidNestedMeta struct {
```

Comment:
You should implements `Is(target error) bool` method for your errors.
It allow callers (the ledger for example) to check for concrete errors.

Reply:
See https://pkg.go.dev/errors